src/compass/utils/h5_helpers.py

```diff
@@ -78,11 +78,13 @@ def add_dataset_and_attrs(group, meta_item):
 
 
 def init_geocoded_dataset(grid_group, dataset_name, geo_grid, dtype,
-                          description):
+                          description, init_value):
     '''
     Create and allocate dataset for isce.geocode.geocode_slc to write to that
     is NC compliant
```

Comment:
Confused about the docstring. Can you be more descriptive? Also, do you mean NC stands for NetCDF?